Frameworks/Foundation/_NSUndoGroup.mm

```diff
@@ -0,0 +1,106 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <Starboard.h>
+#import <Foundation/NSUndoManager.h>
+#import "_NSUndoManagerInternal.h"
+
+@implementation _NSUndoGroup {
+    StrongId<_NSUndoManagerStack> _undoObjectsInGroup;
```

Comment:
>_undoObjectsInGroup; [](start = 34, length = 20)

nit: this is better, but this seems like it can probably be narrowed down to '_undoablesInGroup'? 'undoObjects' just adds ambiguity (is there an undo object that's NOT an NSUndoable, etc)